course/staff_views.py

```diff
@@ -145,7 +146,15 @@ class EnrollStudentsView(CourseInstanceMixin, BaseFormView):
     form_class = EnrollStudentsForm
     template_name = "course/staff/enroll_students.html"
 
+    def get_form_kwargs(self):
+        kwargs = super().get_form_kwargs()
+        kwargs["instance"] = self.instance
+        return kwargs
+
     def form_valid(self, form):
+        if (self.instance.sis_id and form.cleaned_data["sis"]):
+            self.instance.enroll_from_sis()
```

Comment:
The teacher should receive some feedback from the SIS enrollment. E.g., "10 students from SIS have been successfully enrolled in the course." or a warning "No students were found in SIS.".
The message can be shown with `messages.info()` and `messages.warning()`